scene/gui/dialogs.cpp

```diff
@@ -443,7 +443,9 @@ bool AcceptDialog::has_autowrap() {
 void AcceptDialog::register_text_enter(Node *p_line_edit) {
 
 	ERR_FAIL_NULL(p_line_edit);
-	p_line_edit->connect("text_entered", this, "_builtin_text_entered");
+	LineEdit *line_edit = Object::cast_to<LineEdit>(p_line_edit);
+	if (line_edit)
```

Comment:
IMO this should be an error instead, e.g. `ERR_FAIL_NULL(Object::cast_to<LineEdit>(p_line_edit))`.

Reply:
I found similar part of code in which error isn't printed

https://github.com/godotengine/godot/blob/edf2ed5e5056b3f37014cafb2571dd31d65a4a96/editor/editor_inspector.cpp#L1889-L1893

Reply:
Well I'd argue that this one should be changed too. But I guess it doesn't hurt as is per se, it's just less correct.